### ISSUE TITLE
Redundant condition

### ISSUE THREAD

**Issue by ChristopheCVB** [XenoPyax/EDJ#5]
https://github.com/XenoPyax/EDJ/blob/46470f547d00ab12b5d6686be87f50ba99ec2d46/src/io/github/xenopyax/edj/EDJApiBuilder.java#L132

Condition 'l instanceof EventListener' is redundant and can be replaced with a null check

`private List<EventListener> listenerList` so items inside of it are instances of EventListener

**Comment by Timtower** [comment]
Might also be worth mentioning that it is inverted.
Did not run the code if it was an EventListener

**Comment by ChristopheCVB** [comment]
Yeah, I kinda stopped posting issues here as I'm in the middle of a big refactor of this library.
It's a private Github repo for now, but if you're interested I can add you to it ;)

**Comment by Timtower** [comment]
Yeah, started using this library ~30 minutes ago, does what I want, just has some issues.
Would love to help.

**Comment by ChristopheCVB** [comment]
Let me get back home and I'll add you to the project

**Comment by ChristopheCVB** [comment]
@Timtower https://github.com/ChristopheCVB/EliteDangerousAPI
